Packs/GuardiCore/Integrations/integration-GuardiCore.yml

```diff
@@ -772,20 +795,42 @@ script:
 
     delete(args['raw-response']);
 
-
     var token = sendRequest(methodDictionary.login, urlDictionary.login, undefined, {username: params.credentials.identifier, password: params.credentials.password}).access_token;
 
     var commandUrlKey;
 
     switch (command) {
         case 'test-module':
-            sendRequest(methodDictionary.logout, urlDictionary.logout, token);
+            requestLogout(methodDictionary.logout, urlDictionary.logout, token);
             return 'ok';
 
         case 'guardicore-get-incident-pcap':
         case 'guardicore-get-incident-attachments':
             commandUrlKey = 'guardicore-get-incident';
             break;
+        case 'guardicore-get-incidents':
+            var is_date_exist = false
+            var regEx = /^\d{4}-\d{2}-\d{2}$/;
+
+            if (args["to_time"]){
+                is_date_exist = true
+                if (args["to_time"].match(regEx)){
+                    args["to_time"] = args["to_time"] + "T00:00:00Z"
+                } else { throw 'Argument \'to_time\' is not with the right format. Format should be: YYYY-MM-DD'}
+            }
+
+            if (args["from_time"]){
+                is_date_exist = true
+                if (args["from_time"].match(regEx)){
+                    args["from_time"] = args["from_time"] + "T00:00:00Z"
+                } else { throw 'Argument \'from_time\' is not with the right format. Format should be: YYYY-MM-DD'}
+            }
+
+            if (!is_date_exist){
+                throw 'Missing argument \'from_time\' or \'to_time\' '
```

Comment:
you need both `from_time ` and `to_time` to exist, correct?
if so, please revisit the logic, it will be true even if only 1 exists.

Reply:
As I understand from the logs, only one of them is required, that why it wrote like this. 
"description":"Missing parameters 'from_time' or 'to_time' from query params"